### DISCUSSION TITLE
Test tarea

### DISCUSSION THREAD

**Discussion by Agustinuc** [IIC2133-PUC/2024-1#180] [Tarea 3] [answered]
hola, se subiran durante la semana tests para la tarea? o con los que estan en el enunciado basta.Gracias

**Comment by benjavicente** [comment, marked as answer]
Siempre con lo entregado en el enunciado basta 🌚 
De todas formas, subimos más ejemplos de la parte 1.

Los tests referenciales con lo que se va a evaluar, los subiremos más adelante.
